src/Components/PaymentFilter/PaymentFilterContext.php

```diff
@@ -53,4 +56,9 @@ public function getCart(): ?Cart
     {
         return $this->cart;
     }
+
+    public function getFlag(string $key): bool
+    {
+        return $this->flags[$key] ?? false;
```

Comment:
Here I would rather return an `isset($this->flags[$key])`, as it is not defined what type the values in the "flags" array are. Or it would actually make more sense if the "flags" array had no key-value pairs, where the flag is the key, but simply the flags as values. And then you could make an "in_array" here.

Reply:
done